commands/core/init.drush.inc

```diff
@@ -0,0 +1,173 @@
+<?php
```

Comment:
We already have https://github.com/drush-ops/drush/blob/master/lib/Drush/CommandFiles/InitCommandFile.php. We should not add this old style file back in, I think.